src/stories/Library/Modals/modal-find-on-shelf/modal-find-on-shelf.scss

```diff
@@ -4,11 +4,11 @@
   align-items: flex-start;
 
   @include breakpoint-m {
-    padding: 0 215px $s-4xl 215px;
+    padding: $s-4xl $s-7xl;
   }
 
   &__headline {
-    margin: 72px $s-md 0 $s-md;
```

Comment:
Since we're moving the margins up to be padding in the modal, let's add these 72 px up in the .modal-find-on-shelf class in padding: 72px $s-md (or if there is a variable for 72, then that). Because now we are missing mobile padding/margin above the headline as seen on the picture.

![image](https://user-images.githubusercontent.com/28546954/191013309-b5549082-cbb2-4a81-a5fa-629f2b925718.png)